packages/forms-web-app/src/pages/projects/_middleware/_utils/get-vertical-tabs.js

```diff
@@ -32,6 +32,12 @@ function getVerticalTabs(projectName, caseRef) {
 			name: 'Examination timetable',
 			url: '/projects/' + caseRef + '/examination-timetable'
 		},
+		{
+			hidden: featureFlag.allowHaveYourSay != true && !hasOpenTimetables,
```

Comment:
Still thinks this needs to be the 'or'? `featureFlag.allowHaveYourSay != true && !hasOpenTimetables`

Otherwise we are saying the both those conditions need to be met for it to be hidden. Is that what we want?

Reply:
yes.

Then when we remove the flag it will work the same...